### PR TITLE
Restyle of unlock page - made to look similar to hold, publish interactions

### PR DESCRIPTION
**Pull request by Donna-H**
## Changes in this PR:
Restyle of unlock page - made to look similar to hold, publish interactions
## Trello card / Rollbar error (etc)
https://trello.com/c/6SUqtJGI/1444-eui-design-review
## Screenshots of UI changes:
Hold and Publish interactions 
after-put-on-hold-add-back-review-link

after-publish-add-review

Restyle of unlock page - made to look similar to hold, publish interactions
### Before
before-redesgn

### After
Screenshot from 2023-11-09 16-04-38

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by Donna-H** [comment]
[[EUI] Design Review](https://trello.com/c/6SUqtJGI/1444-eui-design-review)

**Comment by Donna-H** [comment]
[[EUI] Design Review](https://trello.com/c/6SUqtJGI/1444-eui-design-review)

File: ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.judgment-unlock {
```
Comment (jacksonj04) [inline ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss:1]: This feels like something which is too specific (ie there's no reason for the unlock page to have its own clsses here); could we make an existing class more generic and deduplicate the rest?

**Review by jacksonj04 — CHANGES_REQUESTED** [review]

File: ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss (hunk 1 above)
Comment (Donna-H) [reply ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss:1]: @jacksonj04 I kept this class from when page was originally created just  reordered the name put judgement first .  The other interactions this is similar to on hold and publish use the class judgment-component so can be changed to that instead and delete the style sheet  judgment_unlock.scss add all styling to the judgment_component.scss which is also used by metadata_panel_static.html and judgment_with-sidebar.html @timcowlishaw any thought on this to change ?

Comment (jacksonj04) [reply ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss:1]: That would be my favourite, there's no reason for the unlock page to have any of its own styling.

Comment (Donna-H) [reply ds_caselaw_editor_ui/sass/includes/_judgment_unlock.scss]: @jacksonj04 updated and screenshots updated too

**Review by jacksonj04 — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**